dmutils/user.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, user_id, email_address, supplier_code, supplier_name,
         self.application_id = application_id
         self.frameworks = frameworks
         self.notification_count = notification_count
+        self.teams = teams
```

Comment:
Should we default `self.teams` to an empty List if it is `None`? I note below in the `from_json()` function we're defaulting to an empty list when instantiating the `User()`. Would it be better to do this here so any falsey `teams` arg value is always an empty list? Doing this will also guard against the possibility below of a `TypeError` if `self.teams` ever makes it to one of the generators below that iterate `self.teams`.

Reply:
defaulted to empty list